### DISCUSSION TITLE
Contribution graph not updating

### DISCUSSION THREAD

**Discussion by nicopicchio** [community/community#66463] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, as mentioned in the title, my contribution graph is not tracking all my commits.

I recently merged this [PR](https://github.com/uktrade/digital-workspace-v2/pull/433/commits) to `main` but my commits are not showing up on my contribution graph and I cannot understand why.
The repo visibility is set to public but I would have the "show contributions to private repos" option enabled anyway.

My local git email is the same as the one registered with my GitHub account.

As you can see from the PR I made the following commits:
| Date | No. of commits |
| :-------: | :---------------: |
| Aug, 22 | 1 |
| Aug, 29 | 6 |
| Aug, 30 | 2 |
| Sep, 4 | 6 |
| Sep, 6 | 7 |
| Sep, 7 | 2 |

But my contribution graph only shows the following:
| Date | No. of commits | Is correct? |
| :---: | :---: |  :---: |
| Aug, 22 | 1 | Yes |
| Aug, 29 | 1 | No |
| Aug, 30 | 0 | No |
| Sep, 4 | 0 | No |
| Sep, 6 | 0 | No |
| Sep, 7 | 1 | No |

My understanding is that commits should not show on work done on branches but once you create and merge a PR you should see the contribution graph filling up based on the dates the commits were made.

Thank you in advance!

**Comment by itsfuad** [comment]
[Common reasons that contributions are not counted](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#common-reasons-that-contributions-are-not-counted)

- When rebasing commits, the original authors of the commit and the person who rebased the commits, whether on the command line or on GitHub.com, receive contribution credit.
- If you merged multiple personal accounts, issues, pull requests, and discussions will not be attributed to the new account and will not appear on your contribution graph.

You can learn more over here [Missing contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)
